lib/livebook_web/live/session_live/app_teams_live.ex

```diff
@@ -296,7 +308,7 @@ defmodule LivebookWeb.SessionLive.AppTeamsLive do
         <div class="flex gap-2 items-center text-gray-700">
           <h3 class="text-sm">
             <span class="font-semibold"><%= @deployment_group.name %></span>
-            (internal-domain.example.com)
+            <.deployment_group_url url={@deployment_group.url} />
```

Comment:
```suggestion
            <span :if={url = @deployment_group.url}>(<%= url %>)</span>
```

Reply:
I don't think those are an issue. We are meant to point to the place the apps are running, so it is all fine. :)